lib/internal/per_context.js

```diff
@@ -5,48 +5,4 @@
 (function(global) {
   // https://github.com/nodejs/node/issues/14909
   if (global.Intl) delete global.Intl.v8BreakIterator;
-
```

Comment:
👆 Should probably check in on the `global.Intl` bit in a follow-up (not this PR) to see if it's still needed too.